libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3258,7 +3226,40 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
       },
     ],
   },
-  // Light Integrations are at the end of the list until we figure out a way to fix the ticker/managerApp collisions
+  avalanche_c_chain: {
+    type: "CryptoCurrency",
+    id: "avalanche_c_chain",
+    coinType: CoinType.ETH,
+    name: "Avalanche C-Chain",
+    // TODO: Do we want to use "Avalanche" instead ?
+    managerAppName: "Ethereum",
+    ticker: "AVAXC",
+    scheme: "avalanche_c_chain",
+    color: "#E84142",
+    family: "ethereum",
+    units: [
+      {
+        name: "AVAX",
+        code: "AVAX",
+        magnitude: 18,
+      },
+    ],
+    ethereumLikeInfo: {
+      baseChain: "mainnet",
+      chainId: 43114,
+      networkId: 43114,
+    },
+    explorerViews: [
+      {
+        tx: "https://cchain.explorer.avax.network/tx/$hash",
+        address: "https://cchain.explorer.avax.network/address/$address",
+        token:
+          "https://cchain.explorer.avax.network/token/$contractAddress?a=$address",
+      },
+    ],
+    keywords: ["avax", "avalanche", "c-chain"],
+  },
+  // FIXME: Light Integrations are at the end of the list until we figure out a way to fix the ticker/managerApp collisions
```

Comment:
IMO this comment should be put at the top of the list, and be reworded to be not specific to "light integrations" (which is unclear btw) as the issue would happen for any multiple currencies using the same `managerAppName`.